src/pages/EditMonitor.vue

```diff
@@ -889,6 +824,36 @@
                                 </div>
                             </div>
 
+                            <!-- Websocket Upgrade only -->
+                            <template v-if="monitor.type === 'websocket-upgrade' ">
+                                <div class="my-3">
+                                    <label for="acceptedStatusCodes" class="form-label">{{ $t("Accepted Status Codes") }}</label>
+
+                                    <VueMultiselect
+                                        id="acceptedStatusCodes"
+                                        v-model="monitor.accepted_statuscodes"
+                                        :options="acceptedWebsocketCodeOptions"
+                                        :multiple="true"
+                                        :close-on-select="false"
+                                        :clear-on-select="false"
+                                        :preserve-search="true"
+                                        :placeholder="$t('Pick Accepted Status Codes...')"
+                                        :preselect-first="false"
+                                        :max-height="600"
+                                        :taggable="true"
+                                    ></VueMultiselect>
+
+                                    <div class="form-text">
+                                        {{ $t("acceptedStatusCodesDescription") }}
+                                    </div>
+                                    <i18n-t tag="div" class="form-text" keypath="wsCodeDescription">
+                                        <template #documentation>
+                                            <a href="https://datatracker.ietf.org/doc/html/rfc6455#section-7.4" target="_blank" rel="noopener noreferrer">{{ 'RFC6455' }}</a>
```

Comment:
not qutie sure why you did it like that, but I would do it like this. 🤔


```suggestion
                                            <a href="https://datatracker.ietf.org/doc/html/rfc6455#section-7.4" target="_blank" rel="noopener noreferrer">RFC6455</a>
```